source/uwp/Renderer/lib/DefaultResourceDictionary.h

```diff
@@ -65,3 +65,129 @@ const PCWSTR c_defaultResourceDictionary = L"\
     <Style x:Key=\"Adaptive.SelectAction\" TargetType=\"Button\" BasedOn=\"{StaticResource  Adaptive.TouchTarget}\"/> \
     <Style x:Key=\"Adaptive.Input.Text.InlineAction\" TargetType=\"Button\" BasedOn=\"{StaticResource  Adaptive.TouchTarget}\"/> \
 </ResourceDictionary>";
+
+
+const PCWSTR c_defaultActionSentimentResourceDictionary = L"\
+<ResourceDictionary \
+    xmlns=\"http://schemas.microsoft.com/winfx/2006/xaml/presentation\" \
+    xmlns:x=\"http://schemas.microsoft.com/winfx/2006/xaml\" \
+    xmlns:dxg=\"http://schemas.devexpress.com/winfx/2008/xaml/grid\" \
+    xmlns:dxgt=\"http://schemas.devexpress.com/winfx/2008/xaml/grid/themekeys\"> \
+\
+    <SolidColorBrush x:Key=\"Adaptive.Action.Positive.Button.Static.Background\" Color=\"#FFDDDDDD\"/> \
+    <SolidColorBrush x:Key=\"Adaptive.Action.Positive.Button.MouseOver.Background\" Color=\"#FFBEE6FD\" /> \
+    <SolidColorBrush x:Key=\"Adaptive.Action.Positive.Button.Foreground\" Color=\"#FFFFFFFF\" /> \
+\
+    <SolidColorBrush x:Key=\"Adaptive.Action.Destructive.Button.Foreground\" Color=\"#FF838383\" /> \
```

Comment:
>FF838383 [](start = 86, length = 8)

These should probably match the default accent and attention colors if not specified in host-config